internal/service/organization/resource_organization.go

```diff
@@ -61,11 +61,30 @@ func Resource() *schema.Resource {
 				Type:     schema.TypeString,
 				Optional: true,
 			},
+			"api_access_list_required": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Computed: true,
+			},
+			"multi_factor_auth_required": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Computed: true,
+			},
+			"restrict_employee_access": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Computed: true,
+			},
 		},
 	}
 }
 
 func resourceMongoDBAtlasOrganizationCreate(ctx context.Context, d *schema.ResourceData, meta any) diag.Diagnostics {
+	if !isAPIKeyOrgOwner(conversion.ExpandStringList(d.Get("role_names").(*schema.Set).List())) {
+		return diag.FromErr(fmt.Errorf("`role_names` for new API Key must have the ORG_OWNER role to use this resource"))
```

Comment:
nit: you could return the err from the method

Reply:
updated